app/views/pages/landing.html.erb

```diff
@@ -1 +1,16 @@
 <h2>landing page</h2>
+
+<br/>
+
+<% if current_user %>
+    <p>Hello, <%= current_user.email %>!</p>
+    <p>This is a placeholder to signify that you have logged in.</p>
+    <p>You are a <%= current_user.role %></p>
+
+    <% if current_user.admin? %>
+        There are now <%= User.count - 1 %> users who have signed up and are waiting your approval.
```

Comment:
`User.count - 1` why less it by 1?

Reply:
ideally, siguro i-change into something like `User.where(is_verified: false)` in the future once implemented

Reply:
got it. 😅